core/packages/contracts/test/BeefyClient.t.sol

```diff
@@ -413,4 +415,36 @@ contract BeefyClientTest is Test {
         vm.expectRevert(BeefyClient.NotEnoughClaims.selector);
         beefyClient.submitInitial(commitment, initialBits, finalValidatorProofs[0]);
     }
+
+    function testSubmitFailWithIncorrectValidatorSetLengthOnTicket() public {
+        BeefyClient.Commitment memory commitment = initialize(setId);
+
+        beefyClient.submitInitial(commitment, bitfield, finalValidatorProofs[0]);
+
+        beefyClient.setTicketValidatorSetLen(keccak256(beefyClient.encodeCommitment_public(commitment)), setSize / 2);
```

Comment:
Manually editing tickets like this isn't a great way to test - we should rather have test data with different validator set lengths. But let's get the updated tests & fixture data in #901 merged first.

Reply:
Already merged.